clearwater-infrastructure/usr/share/clearwater/bin/clearwater-show-config

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# @file clearwater-show-config
```

Comment:
Thanks for the heads up. I am going to agree whether we use this script, or the one it conflicts with, or some combination of the two.